pkg/controller/machine/machine.go

```diff
@@ -53,12 +45,20 @@ import (
 	"k8s.io/client-go/tools/reference"
 	"k8s.io/client-go/util/retry"
 	"k8s.io/client-go/util/workqueue"
-
 	"sigs.k8s.io/cluster-api/pkg/apis/cluster/common"
 	clusterv1alpha1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
 	clusterv1alpha1clientset "sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset"
 	machinescheme "sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/scheme"
 	clusterlistersv1alpha1 "sigs.k8s.io/cluster-api/pkg/client/listers_generated/cluster/v1alpha1"
+
+	"github.com/kubermatic/machine-controller/pkg/cloudprovider"
```

Comment:
While splitting our own imports from the third party imports into an own group is a good thing, we usually order it like this:

```
Go SDK
external packages which do not apply to other rules (Like github.com/golang/glog, etc.)
github.com/kubermatic
k8s.io/*
```

Ref: https://github.com/kubermatic/kubermatic/blob/9c59130831f63a2ebe703d7ae328fd9459112d7a/docs/code-style.md#imports

Reply:
Good to know, shout have a code style guide document for it. Only know the general rule

```
import (
    stdlib

    3rd party packages

    own packages
)
```

and don't know why importing k8s should be treated differently. We often even mix own and k8s packages in imports.

Reply:
The code style document is linked above